server/etcdserver/api/v3discovery/discovery.go

```diff
@@ -0,0 +1,542 @@
+// Copyright 2022 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package v3discovery provides an implementation of the cluster discovery that
+// is used by etcd with v3 client.
+package v3discovery
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"go.etcd.io/etcd/client/pkg/v3/transport"
```

Comment:
Can you fix import order? In this file and others.

Reply:
It's added automatically and gofmted. It seems gofmt isn't always making the correct decision. 
Will fix.

Reply:
all fixed.